backend/app/Http/Controllers/UserController.php

```diff
@@ -36,4 +36,12 @@ public function login(Request $request)
 
         return LoginResource::make(array_merge($user->toArray(), ['token' => $token]));
     }
+    public function myAccounts(Request $request)
```

Comment:
space 
```suggestion
    }
    
    public function myAccounts(Request $request)
```

Reply:
@IanUrriza10 pls also change the function name, `myAccounts` doesnt seem to fit with other relationship function names